server/api/account.js

```diff
@@ -471,6 +489,16 @@ module.exports.init = function(server) {
             user.password = await hashPassword(userToSet.password, 10);
         }
 
+        if(userToSet.enableGravatar && !user.enableGravatar) {
+            user.emailHash = crypto.createHash('md5').update(user.email).digest('hex');
+        } else if(!userToSet.enableGravatar && user.enableGravatar) {
+            user.emailHash = DefaultEmailHash;
```

Comment:
Since we use the same image for all users without Gravatar enabled, would it make more sense to download that once locally, and then copy it over to the new location as updates are made? The current flow re-downloads it from Gravatar each time.

Reply:
It is slightly wasteful, but at the volumes we're using I think it will be fine

I plan to run a script to update the existing users en masse